guide/the-scene-tree.md

```diff
@@ -30,6 +30,8 @@ value.
 ![](images/help-button.png =26x26) `Help`: Context sensitive help for the
 currently selected node.
 
+![](images/focus_button.png =26x26) `Focus`: Move viewpoint to focus on selected node.
```

Comment:
Moving the viewpoint actually moves the scene ?
Like saying the world moves and not the viewer ?